packages/fhir-client/src/components/PatientDetails/ResourceSchema/Patient.tsx

```diff
@@ -73,7 +73,7 @@ export const columns = (t: TFunction) =>
       // eslint-disable-next-line react/display-name
       render: (record: PatientTableData) => (
         <span className="d-flex justify-content-start align-items-center">
-          <Link to={`${LIST_PATIENTS_URL}/${record.id}`} className="m-0 p-1">
+          <Link to={`${LIST_PATIENTS_URL}?${viewDetailsQuery}=${record.id}`} className="m-0 p-1">
```

Comment:
nit: prefer use of [urlParams](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/URLSearchParams#examples) when creating the query string